### PR TITLE
debugpb: add GetAllRegionsInStore()

### PR DESCRIPTION
**Pull request by andylokandy**
Signed-off-by: Andy Lok <andylokandy@hotmail.com>

### CODE REVIEW

File: proto/debugpb.proto [hunk 1]
```diff
@@ -248,3 +251,10 @@ message GetClusterInfoRequest {
 message GetClusterInfoResponse {
     uint64 cluster_id = 1;
 }
+
+message GetAllRegionsInStoreRequest {
+}
+
+message GetAllRegionsInStoreResponse {
+    repeated uint64 regions = 1;
```
Comment (disksing) [inline proto/debugpb.proto:259]: Is id enough? I think we may add more information for debug purpose.

Comment (andylokandy) [reply proto/debugpb.proto:259]: The `RegionInfo` rpc should be able to return the detailed information for each regon.

Comment (disksing) [reply proto/debugpb.proto:259]: ok

**Review by disksing — APPROVED** [review]

**Review by Connor1996 — APPROVED** [review]
LGTM

**Pull request merged**